internal/db/schema/migrations/oss/postgres/60/02_add_ssh_cert_to_vault_cred_library_view.up.sql

```diff
@@ -0,0 +1,112 @@
+    store.project_id          as project_id,
+    store.vault_address       as vault_address,
+    store.namespace           as namespace,
+    store.ca_cert             as ca_cert,
+    store.tls_server_name     as tls_server_name,
+    store.tls_skip_verify     as tls_skip_verify,
+    store.worker_filter       as worker_filter,
+    store.ct_token            as ct_token, -- encrypted
+    store.token_hmac          as token_hmac,
+    store.token_status        as token_status,
+    store.token_key_id        as token_key_id,
+    store.client_cert         as client_cert,
+    store.ct_client_key       as ct_client_key, -- encrypted
+    store.client_key_id       as client_key_id,
+    coalesce(upasso.username_attribute,sshpk.username_attribute)
+      as username_attribute,
+    upasso.password_attribute              as password_attribute,
+    sshpk.private_key_attribute            as private_key_attribute,
+    sshpk.private_key_passphrase_attribute as private_key_passphrase_attribute,
+    'generic'                              as cred_lib_type -- used to switch on
+    from credential_vault_library library
+    join credential_vault_store_client store
+      on library.store_id = store.public_id
+    left join password_override upasso
+      on library.public_id = upasso.library_id
+    left join ssh_private_key_override sshpk
+      on library.public_id = sshpk.library_id
+  union
+  select library.public_id   as public_id,
+    library.store_id         as store_id,
+    library.name             as name,
+    library.description      as description,
+    library.create_time      as create_time,
+    library.update_time      as update_time,
+    library.version          as version,
+    library.vault_path       as vault_path,
+    null                     as http_method,
+    null                     as http_request_body,
+    library.credential_type  as credential_type,
+    library.key_type         as key_type_attribute,
+    library.key_bits         as key_bits_attribute,
+    library.ttl              as ttl_attribute,
+    library.key_id           as key_id_attribute,
+    library.critical_options as critical_options_attribute,
+    library.extensions       as extensions_attribute,
+    store.project_id         as project_id,
+    store.vault_address      as vault_address,
+    store.namespace          as namespace,
+    store.ca_cert            as ca_cert,
+    store.tls_server_name    as tls_server_name,
+    store.tls_skip_verify    as tls_skip_verify,
+    store.worker_filter      as worker_filter,
+    store.ct_token           as ct_token, -- encrypted
+    store.token_hmac         as token_hmac,
+    store.token_status       as token_status,
+    store.token_key_id       as token_key_id,
+    store.client_cert        as client_cert,
+    store.ct_client_key      as ct_client_key, -- encrypted
+    store.client_key_id      as client_key_id,
+    library.username         as username_attribute,
```

Comment:
As mentioned in the go code, this shouldnt reuse `username_attribute` instead there should be a new column added for just `username`.